MoonCrystal/MoonCrystal/FormatInputView/HighFormatRadioButton.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  HighFormatRadioButton.swift
+//  MoonCrystal
+//
+//  Created by sungkug_apple_developer_ac on 7/31/24.
+//
+
+import SwiftUI
+
+struct HighFormatRadioButton: View {
+    @Binding var selectedType: VideoFormatCapacity?
+    var format: VideoFormatCapacity
+
+    var body: some View {
+        HStack {
+            Button(action: {
+                selectedType = format
+            }, label: {
+                HStack {
+                    Circle()
+                        .stroke(selectedType == format ? Color.pink300 : Color.gray, lineWidth: 1)
+                        .frame(width: 18, height: 18)
+                        .overlay(
+                            Circle()
+                                .fill(selectedType == format ? Color.pink300 : Color.white)
+                                .frame(width: 12, height: 12)
+                        )
+                    Text(format.detailInfo)
+                        .foregroundColor(.black)
+                        .font(.system(size: 16, weight: selectedType == format ? .bold : .regular))
+                }
+            })
```

Comment:
요렇게 쓰는 것은 어떻게 생각하시나요 ??
```suggestion
            Button {
                selectedType = format
            } label: {
                HStack {
                    Circle()
                        .stroke(selectedType == format ? Color.pink300 : Color.gray, lineWidth: 1)
                        .frame(width: 18, height: 18)
                        .overlay(
                            Circle()
                                .fill(selectedType == format ? Color.pink300 : Color.white)
                                .frame(width: 12, height: 12)
                        )
                    Text(format.detailInfo)
                        .foregroundColor(.black)
                        .font(.system(size: 16, weight: selectedType == format ? .bold : .regular))
                }
            }
```

Reply:
아하 자동완성 쓰다보니 이렇게 되었네요 이건 수정해두겠습니다!